controllers/ibmpowervsimage_controller.go

```diff
@@ -97,8 +92,9 @@ func (r *IBMPowerVSImageReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 		return r.reconcileDelete(imageScope)
 	}
 
+	cluster, err := util.GetClusterByName(ctx, r.Client, ibmImage.Namespace, ibmImage.Spec.ClusterName)
 	if err != nil {
-		return reconcile.Result{}, errors.Errorf("failed to create scope: %+v", err)
+		return ctrl.Result{}, err
```

Comment:
any specific reason for using from the ctrl module? wondering if we can keep it sync with other returns in the program.

Reply:
No specific reason , I did not notice, it is being [there](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/main/controllers/ibmpowervsimage_controller.go#L68-L75) already, May be I will update to use `reconcile.Result{}`

Reply:
Just updated one occurance of reconcile.Result{} to ctrl.Result{} to keep it consistent in the file